### DISCUSSION TITLE
第4节课作业(5班)

### DISCUSSION THREAD

**Discussion by Farewell-CK** [InternLM/Tutorial#325] [作业]
宝子们，五班的第四次作业交这里哦

**Comment by DreamBinary** [comment]
https://github.com/DreamBinary/InternLMCamp/tree/master/day04/work

**Comment by shengmouget** [comment]
[第四节课作业](https://github.com/shengmouget/LLM_Job?tab=readme-ov-file#%E7%AC%AC%E4%B8%89%E6%AC%A1%E4%BD%9C%E4%B8%9A)

**Comment by ZhaoHongSen0726** [comment]
https://blog.csdn.net/m0_75085274/article/details/135571657?spm=1001.2014.3001.5501

**Comment by GitEasonXu** [comment]
https://github.com/GitEasonXu/InternLM/blob/main/homework/course_4/coursework.md

**Comment by JayceNing** [comment]
**基础作业：**
构建数据集，使用 XTuner 微调 InternLM-Chat-7B 模型, 让模型学习到它是你的智能小助手，效果如下图所示，本作业训练出来的模型的输出需要将不要葱姜蒜大佬替换成自己名字或昵称！

微调后（对自己的身份有了清晰的认知） 
![image](https://github.com/InternLM/tutorial/assets/44694099/1575c288-3766-4c74-9279-b3e41d246522)

**Comment by ATang0729** [comment]
https://github.com/ATang0729/InternLMCamp_Notes/blob/main/notes/class4-HW/class4_HW.md

**Comment by Milkve** [comment]
[第四次作业](https://juejin.cn/spost/7324253082615889971)

**Comment by angellyao** [comment]
第四次作业，https://zhuanlan.zhihu.com/p/678190450

**Comment by Farewell-CK** [comment]
第四次作业https://github.com/8baby8/Farewell_Learn_puyu/blob/main/%E7%AC%AC%E5%9B%9B%E8%8A%82%E8%AF%BE/task.md

**Comment by sanbuphy** [comment]
https://zhuanlan.zhihu.com/p/680168238?
